devtools-ci-index.yaml

```diff
@@ -4045,13 +4054,29 @@
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: cvejob
             discarder_days: 30
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8io
+            git_repo: kubernetes-model
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_release.sh'
+            timeout: '30m'
+            wrappers:
```

Comment:
you can't put wrapper here, it should be defined in job-template or in a full job definition, sorry I was not clear about that